lib/modules/platform/scmm/types.ts

```diff
@@ -0,0 +1,144 @@
+export type Page<T> = {
```

Comment:
please use zod schema

Reply:
For every type or just this one?

Reply:
Use it whenever parsing data, either from repo files or remote datasource APIs

Reply:
But I am only using the `Page<T>` to fetch data from the SCM-Manager API and not from repo files or remote datasource APIs. I am using it in the same way like `PagedResult` in the Bitbucket implementation. Should I still use the zod schema for the `Page<T>` type?

Reply:
Yes

Reply:
Bitbucket is now using zod schema too:
https://github.com/renovatebot/renovate/blob/f39568d8a809a3b170ecb812f21e38c1689e1ce0/lib/modules/platform/bitbucket/schema.ts#L15-L24